composer.json

```diff
@@ -18,7 +18,7 @@
     },
     "require-dev": {
         "ext-intl": "*",
-        "phpunit/phpunit": "^7.5 || ^8.5 || ^9.4",
+        "symfony/phpunit-bridge": "^5.0",
```

Comment:
Better go with `^5.1`.

Reply:
You could even use `dev-master` for this one which is a bit special (the master version support even very old PHPUnit and PHP versions).

Reply:
Alright, will replace it with `*`. I don't think that we need to support those very old phpunit versions as this PR would change the php version support to `>=7.1`

Reply:
I mean "even" the master version supports very old PHPUnit versions but that's ok.